packages/cxl-ui/src/components/cxl-star-rating.js

```diff
@@ -126,7 +126,6 @@ export class CXLStarRatingElement extends LitElement {
     }
 
     this._saveState(value);
-    this._updateReadonlyAndValue();
```

Comment:
What if you just remove these 2 lines and keep methods in the component?
I have a feeling `_saveState()` is not needed in our case too. Please check it.

Reply:
You're right.